client/src/components/ratingsReviews/RatingBreakdown.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const RatingBreakdown = (props) => {
+  const { ratings, recommended } = props;
+  const stars = [1, 1, 1, .8, 0];
+  let key = 0;
+  console.log('ratings:', ratings);
+
+  if (!recommended.false) {
+    var averageRecommend = 100;
+  } else {
+    var averageRecommend = Math.round(100 / (parseInt(recommended.false) + parseInt(recommended.true)) * parseInt(recommended.true));
```

Comment:
A suggestion for easier to read code for the math involved would be to calculate the average and then multiply by 100 to get the %, such as:

`Math.round(parseInt(recommended.true)/(parseInt(recommended.false) + parseInt(recommended.true)) * 100)`